config/locales/server.zh_CN.yml

```diff
@@ -45,7 +45,7 @@ zh_CN:
         topic_not_found_error: "当回复到已经删除的相关主题时发生。"
         topic_closed_error: "当回复到已经关闭的相关主题时发生。"
   errors: &errors
-    format: '%{attribute} %{message}'
+    format: '%{attribute}%{message}'
```

Comment:
I have updated the translation on Transifex. Here I leave it as an example for the topic on Meta to show how it can be tweaked (translated)